plugins/slack/app/models/slack_notification.rb

```diff
@@ -4,18 +4,20 @@ def initialize(deploy)
     @stage = deploy.stage
     @project = @stage.project
     @user = @deploy.user
+    @webhook = @stage.slack_webhooks.first
```

Comment:
I just realized it's the same as the old code... I think that's a bug. Unless this whole thing should be singleton.